galaxy/worker/importers/apb.py

```diff
@@ -17,9 +17,36 @@
 
 from . import base
 
+from galaxy.main import models
+
 
 class APBImporter(base.ContentImporter):
+    MAX_TAGS_COUNT = 20
```

Comment:
IIRC `MAX_TAGS_COUNT` is used in role parser. Since it's more than one place we use it, please move it to constants.

Reply:
Done.